Source/CesiumRuntime/Private/CesiumGltfPrimitiveComponent.h

```diff
@@ -105,16 +106,118 @@ class UCesiumGltfPrimitiveComponent : public UStaticMeshComponent {
 
   std::optional<Cesium3DTilesSelection::BoundingVolume> boundingVolume;
 
-  /**
-   * Updates this component's transform from a new double-precision
-   * transformation from the Cesium world to the Unreal Engine world, as well as
-   * the current HighPrecisionNodeTransform.
-   *
-   * @param CesiumToUnrealTransform The new transformation.
-   */
-  void UpdateTransformFromCesium(const glm::dmat4& CesiumToUnrealTransform);
+  void BeginDestroyPrimitive();
```

Comment:
This could use a better name. The overridden method on the actual UObjects has to be called `BeginDestroy` to hook into Unreal, but here the `Begin` part doesn't make much sense. Something like `Destroy` or `Clear` would be fine IMO.